src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eo pipefail
+
+repo_dir="$(realpath "$(dirname "$0")/../../..")"
+pot_file="$repo_dir/locales/messages.pot"
```

Comment:
So unsure, but the file name of these .pot's could change from all being messages.pot depending on what @LarissaLane proves out around Transifex projects containing multiple repos.  Just putting that out there.  Given that change will need to permeate this library, probably no need to think about that until the constraints are dialed in.  That in and of itself though, would be a major breaking change/version bump...